tests/conftest.py

```
@@ -14,13 +16,64 @@ def remove_headers(response: dict):
         response["headers"] = {}
         return response
 
+    def scrub_requests(request: dict):
+        if request.body:
+            try:
+                body_str = request.body.decode("utf-8")
+                body_json = json.loads(body_str)
+                if "token" in body_json:
+                    body_json["token"] = "ntn_..."
+                if "code" in body_json:
+                    body_json["code"] = "..."
+                if "redirect_uri" in body_json:
+                    body_json["redirect_uri"] = "http://..."
+                request.body = json.dumps(body_json).encode("utf-8")
+            except json.JSONDecodeError as e:
+                raise json.JSONDecodeError(
+                    f"Failed to decode request body: {request.body} \n Error occurred at {e.pos} with message: {e.msg}",
+                    request.body,
+                    e.pos,
+                )
```

Why not just let it raise? What's the point of the try-except construct here?